stream-chat-android-ui-components/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -615,19 +621,22 @@ public class MessageListViewModel(
                     ?.message
 
                 if (message != null) {
-                    _targetMessage.value = message!!
+                    logger.e { "Function -> LocalMessage" }
+                    focusMessage(message)
                 } else {
-                    chatClient.loadMessageById(
-                        cid,
-                        event.messageId
-                    ).enqueue { result ->
-                        if (result.isSuccess) {
-                            _targetMessage.value = result.data()
-                        } else {
-                            val error = result.error()
-                            logger.e { "Could not load message: ${error.message}. Cause: ${error.cause?.message}" }
+                    ChatClient.instance().getMessageUsingCache(event.messageId)
```

Comment:
Why don't we use the injected instance of `ChatClient`?? :thinking:

Reply:
Good catch.

No reason, just a reflex mistake, pushed the suggested change now. :smile: